configs/vision/pathology/online/segmentation/monusac.yaml

```diff
@@ -81,12 +81,19 @@ model:
         - class_path: torchmetrics.ClasswiseWrapper
           init_args:
             metric:
-              class_path: eva.vision.metrics.GeneralizedDiceScore
+              class_path: eva.vision.metrics.MonaiDiceScore
               init_args:
+                include_background: true
                 num_classes: 6
-                weight_type: linear
+                reduction: none
                 ignore_index: *IGNORE_INDEX
-                per_class: true
+            labels:
+              - background
+              - epithelial
+              - lymphocyte
+              - neutrophil
+              - macrophage
+              - ambiguous
```

Comment:
also here, num classes is inconsistent with documentation. https://kaiko-ai.github.io/eva/main/datasets/monusac/ If doc is wrong can you fix it or create a separate issue? (maybe we should also mention the classes in the doc)

Reply:
updated the docs